app/models/mixins/authentication_mixin.rb

```diff
@@ -386,6 +387,19 @@ def raw_change_password(_current_password, _new_password)
     raise NotImplementedError, _("must be implemented in subclass.")
   end
 
+  def assign_nested_endpoints(attributes)
```

Comment:
Minor but this is assigning a single endpoint so `assign_nested_endpoint` would be more accurate